### DISCUSSION TITLE
Daily 02/10

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#164] [Daily]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by BraianTroncoso** [comment]
Ayer me tome el dia, Hoy me pongo a ver el tema del back con mysql y a leer un poco de documentacion para ya empezar con las tareas.

**Comment by marceloedelgado** [comment]
Domingo break. 

Hoy Lunes comencé a ver las tareas que tengo asignadas para la Sprint 2 (funcionalidades) y las tareas que continuan en la Sprint 2 con la activity_catalogo.xml [Crear más cardsview de profesionales y aplicar un cambio de estructura mediante Recycler view y hacer scrolling vertical.

Dificultad: Para aplicar Recycler view necesitaré crear otras dependencias y lo vengo estudiando y probando en un entorno externo al proyecto para comprender bien la nueva estructura

**Comment by NataliaAlvarezIspc** [comment]
El viernes pudimos, a pesar de las dificultades y las consigna agregadas a último momento, cerrar el sprint 1. El sabado por la mañana tuvimos nuestar review y planing. El resto del sabado y domingo me lo tomé para descansar. 
Hoy estuve revisando el tablero kanban y acomodando las tareas para que se entienda cuales son las que debemos asignarnos para este sprint. También estuve averiguando como hacer que las activities sean responsive y viendo las presentaciones de otros grupos para ver como venimos con respecto al resto. Empecé a hacer los apuntes para la reunión del domingo pero los terminaré mañana. Además revisé los contenidos de la plataforma para ver si habián subido algo con respecto al lo que piden para este sprint pero no encontré nada todavía.
Dificultades. Me cuesta bastante entender la parte de back, así que tendré que dicarle más tiempo por día para entenderlo y poder implementarlo en el proyecto.
